lib/matplotlib/_constrained_layout.py

```diff
@@ -73,11 +73,11 @@ def do_constrained_layout(fig, h_pad, w_pad,
         `.Figure` instance to do the layout in.
 
     h_pad, w_pad : float
-      Padding around the axes elements in figure-normalized units.
+      Padding around the Axes elements in figure-normalized units.
 
     hspace, wspace : float
        Fraction of the figure to dedicate to space between the
-       axes.  These are evenly spread between the gaps between the axes.
+       axes.  These are evenly spread between the gaps between the Axes.
```

Comment:
```suggestion
       Axes.  These are evenly spread between the gaps between the Axes.
```

This one is a bit borderline, because I think this is actually referring to the plural of `Axes`, not the plural of `Axis`, but regardless I think the same sense is meant by both instances on this line, so they should probably be the same at least.